src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -857,6 +870,10 @@ void _model_PropertyChanged(object sender, PropertyChangedEventArgs e)
                         this.PublishCurrentWorkspaceCommand.RaiseCanExecuteChanged();
                     RaisePropertyChanged("IsPanning");
                     RaisePropertyChanged("IsOrbiting");
+                    if (ChangeScaleFactorCommand != null)
```

Comment:
The *CanExecute* status of this new command still does not change properly in some situations (e.g. when the workspace is changed but there is no call to `RaiseProeprtyChanged("CurrentWorkspace")`. Will investigate for a fix after this PR's functionality has been confirmed.